src/site/content/en/metrics/lcp/index.md

```diff
@@ -357,7 +357,7 @@ the article on [custom metrics](/custom-metrics/#element-timing-api).
 
 ## How to improve LCP
 
-LCP is primarily affected by three factors:
+LCP is primarily affected by four factors:
```

Comment:
```suggestion
LCP is primarily affected by the following factors:
```
How about we avoid the problem in the future by avoiding any hardcoded reference to a specific number of factors?

Reply:
Absolutely, great idea.